modules/distr-tracing-tempo-configuring-tempooperator-metrics-and-alerts.adoc

```diff
@@ -1,6 +1,6 @@
 // Module included in the following assemblies:
 //
-// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+// * observability/distr_tracing/distr-tracing-tempo-configuring.adoc
```

Comment:
@max-cx The file is renamed here for your meta data, but I'm not sure if servability/distr_tracing/distr-tracing-tempo-configuring.adoc is actually an assembly in this version of your docs.